conf/application.conf

```diff
@@ -135,7 +135,7 @@ microservice {
 scheduling {
   online-testing {
     send-invitation-job {
-      enabled = false
+      enabled = true
```

Comment:
We keep all the schedulers in dev off by default - so please amend this to false before putting up for review